test/DynamoCoreWpfTests/PreferencesViewModelTests.cs

```diff
@@ -136,5 +139,30 @@ public void TestGetTransformedHostUnits()
             Assert.AreEqual(ViewModel.PreferencesViewModel.GetTransformedHostUnits(Dynamo.Configuration.Configurations.Units.Miles),
                 Dynamo.Configuration.Configurations.Units.Miles);
         }
+
+        [Test]
+        public void PathManagerWithDifferentHostTest()
+        {
+            PathManager singletonPathManager = PathManager.Instance;
+
+            TestPathResolverParams revitResolverParams = new TestPathResolverParams()
+            {
+                UserDataRootFolder = @"C:\\Users\\user\\AppData\\Roaming\\Dynamo\\Dynamo Revit",
+                CommonDataRootFolder = @"C:\\ProgramData\\Autodesk\\RVT 2024\\Dynamo"
+
+            };
+
+            IPathResolver revitPathResolver = new TestPathResolver(revitResolverParams);
+            string dynamoRevitHostPath = @"C:\Program Files\Autodesk\Revit 2024\AddIns\DynamoForRevit\Revit)";
+            singletonPathManager.AssignHostPathAndIPathResolver(dynamoRevitHostPath, revitPathResolver);
+
+            string dynamoRevitUserDataDirectory = "C:\\Users\\user\\AppData\\Roaming\\Dynamo\\Dynamo Revit\\2.19";
```

Comment:
be consistent - IE use a verbatim string unless there is a good reason not to as you did above.